content/search-fitness-activities.md

```diff
@@ -52,6 +62,9 @@ sort_by : _optional_ **string**<br/>
 Sort results by distance_in_meters, duration_in_seconds, elevation_in_meters or
 calories. Default: distance_in_meters
 
+limit : _optional_ **integer**<br/>
```

Comment:
...wondering if this guy should be `Proper Case`, given the mention of `Integer` twice above? (...happy to be told that no one cares ;P)